src/twister2/device/native_simulator_adapter.py

```diff
@@ -1,19 +1,21 @@
 from __future__ import annotations
 
+import asyncio.subprocess
```

Comment:
Can you add a comment explaining why we need asyncio here?

Reply:
I will update the code